packages/common-library/src/common_library/basic_types.py

```diff
@@ -1,4 +1,13 @@
 from enum import StrEnum
+from typing import Any
+
+from pydantic_core import PydanticUndefined
+
+# SEE https://github.com/fastapi/fastapi/blob/master/fastapi/_compat.py#L75-L78
+Undefined = PydanticUndefined
+UNSET: Any = Undefined
```

Comment:
Maybe as you suggested let's call it `DefaultFactory` to avoid the other `UnSet`we now have in the common_library